earth2mip/inference_medium_range.py

```diff
@@ -155,15 +155,17 @@ def process(initial_time):
             # select first history level
             verification_torch = verification_torch[:, -1]
             for metric in metrics:
-                outputs = metric.call(verification_torch, data)
+                indexes = [i for i,name in enumerate(model.out_channel_names) if name in model.in_channel_names]
```

Comment:
I think this line needs to be formatted. When installing earth2mip use:

`pip install .[dev]` or `pip install -e .[dev]` 

This will install dependencies for development. You can manually run formatting / linting using `make format` and `make lint`.

Just for future reference. We also use pre-commit to run these checks ahead of time, inside your earth2mip repo:

```bash
pip install pre-commit
pre-commit install
```